### PR TITLE
Disable iOS PR checks with GitHub Actions in favor of checks on Teamcity

### PR DESCRIPTION
**Pull request by ASalavei**
Describe proposed changes and the issue being fixed

## Release Notes
Disable iOS PR checks using GitHub Actions in favor of Teamcity checks.

### CODE REVIEW

File: .github/workflows/ios-pr.yml [hunk 1]
```diff
@@ -1,10 +1,6 @@
 name: iOS Unit tests
 on:
   workflow_dispatch:
-  pull_request:
```
Comment (igordmn) [inline .github/workflows/ios-pr.yml:4]: I would remove the pipelines completely, as we won't use them.

Comment (ASalavei) [reply .github/workflows/ios-pr.yml:4]: @igordmn , I made it to be aligned with other pipelines. They also aren't active and have manual `workflow_dispatch` launch trigger. The decision is up to you, so what we can do here:
- Merge current MR with no changes.
- Remove only `ios-pr.yml'.
- Remove all pipelines.

Comment (igordmn) [reply .github/workflows/ios-pr.yml:4]: Others are either "temporary" or came from AOSP.

let's remove `ios-pr.yml`

Comment (ASalavei) [reply .github/workflows/ios-pr.yml:4]: Done

**Review by igordmn — APPROVED** [review]

**Pull request merged**